src/game/server/gamemodes/instagib/foot/sql_columns.h

```diff
@@ -0,0 +1,8 @@
+// This file can be included several times.
+
+#ifndef MACRO_ADD_COLUMN
+#error "The column macros must be defined"
+#define MACRO_ADD_COLUMN(name, sql_name, sql_type, bind_type, default, merge_method) ;
+#endif
+
+MACRO_ADD_COLUMN(WrongGoal, "wrong_goal", "INTEGER", Int, "0", Add)
```

Comment:
Go plural like all other columns.

```suggestion
MACRO_ADD_COLUMN(WrongGoals, "wrong_goals", "INTEGER", Int, "0", Add)
```